content/docs/command-reference/data/status.md

```diff
@@ -83,31 +88,6 @@ option.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
```

Comment:
why removing it?

Reply:
I suggest removing it because it's already used in the description.

> You asked me to make a PR with the most important changes from my feedback and IMO this is one of them: https://github.com/iterative/dvc.org/pull/3812#discussion_r963326344

Reply:
https://github.com/iterative/dvc.org/pull/3812#discussion_r958632037 - the problem is it was discussed already. Question - do you have an actually strong opinion about this? :) (I'm personally fine and Dave was fine it seems)

Reply:
I didn't realize it had been discussed (I didn't read all the resolved comments). I still think it's unnecessary since the `--granular` example can still refer to the one in the description.

It's not a auper strong opinion but it does go against good practices for cmd ref examples, I think: they should add special value to the doc, not just cover obvious cases. And to avoid redundancy in general.

But anyway, I rolled it back in 725c7e4f3 since it was discussed by them.